platform/openide.util/src/org/openide/util/MapFormat.java

```diff
@@ -276,12 +282,16 @@ public StringBuffer format(Object pat, StringBuffer result, FieldPosition fpos)
 
             if (obj == null) {
                 if (throwex) {
-                    throw new IllegalArgumentException(
-                        MessageFormat.format(
-                            NbBundle.getBundle(MapFormat.class).getString("MSG_FMT_ObjectForKey"),
-                            new Object[] { new Integer(key) }
-                        )
-                    );
+                    throw new IllegalArgumentException() {
+                        @Override
+                        public String getMessage() {
+                            // this is ugly- but it allows unit testing
```

Comment:
Why?

Reply:
Executing
NbBundle.getBundle(MapFormat.class).getString("MSG_UnmatchedBraces")
throws exception saying thet no bundle has been found.
so I detered this execution to be able to run tets

better solution is to make sure bundle exists while unit tests are being executed - I don't know how to do it yet